snmp/conf.yaml.example

```diff
@@ -75,6 +75,8 @@ instances:
   #   privKey: private_key
   #   authProtocol: authProtocol
   #   privProtocol: privProtocol
+  #   context_engine_id: contextEngineId  # optional v3-only parameter - typically unneeded
```

Comment:
Nit: should we add these to the `SNMP_V3_CONF` test configuration? That would increase coverage a bit.

Reply:
I'll try to add that to the CI - it involves modifying the actual CI setup, that's why I skipped it  (as It's an optional parameter in SNMP v3).